### PR TITLE
CI: adjust doctests for lightning.pytorch

### PR DESCRIPTION
**Pull request by awaelchli**
## What does this PR do?

Same as  #15457 but for doctest. See the failing run https://github.com/Lightning-AI/lightning/actions/runs/3373154980/jobs/5597446136 in PR #15301

## Before submitting
- [x] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes? (if necessary)
- [x] Did you write any new necessary tests? (not for typos and docs)
- [x] Did you verify new and existing tests pass locally with your changes?
- [x] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?
I made sure I had fun coding 🙃

cc @carmocca @akihironitta @borda

### CODE REVIEW

**Review by otaj — APPROVED** [review]

**Review by Borda — APPROVED** [review]

File: .github/workflows/docs-checks.yml [hunk 1]
```diff
@@ -79,6 +79,10 @@ jobs:
           pip install -e . -U  --find-links https://download.pytorch.org/whl/cpu/torch_stable.html --find-links pypi
           git checkout -- setup.py MANIFEST.in
 
+      - name: Adjust examples
+        if: ${{ matrix.pkg-name == 'lightning' }}
+        run: python .actions/assistant.py copy_replace_imports --source_dir="./src" --source_import="pytorch_lightning,lightning_lite" --target_import="lightning.pytorch,lightning.lite"
```
Comment (Borda) [inline .github/workflows/docs-checks.yml:84]: just thinking now that this shall be replaced in docs, right?

**Comment by awaelchli** [comment]
Why did you merge this :(I disabled auto merge. It is not working, as you can see I was testing it here: https://github.com/Lightning-AI/lightning/pull/15301

File: .github/workflows/docs-checks.yml (hunk 1 above)
Comment (awaelchli) [reply .github/workflows/docs-checks.yml:84]: @Borda I have no idea what I'm doing, just trying to unblock us from merging to master.

**Comment by Borda** [comment]
> Why did you merge this :(I disabled auto merge. It is not working, as you can see I was testing it here: #15301

it was my bad I checked and seems fine, did not realize the names as the GH's diff was too small... my had :(

**Pull request merged**